packages/ecs-agent/ecs-agent.spec

```
@@ -289,6 +314,8 @@ mv %{vpccni_gorepo}-%{vpccni_gitrev}/vendor go-vendor/%{vpccni_gorepo}
 %{_cross_libexecdir}/amazon-ecs-agent/ecs-eni
 %{_cross_libexecdir}/amazon-ecs-agent/ecs-ipam
 %{_cross_libexecdir}/amazon-ecs-agent/vpc-branch-eni
+%{managed_agents}
+%{ecs_exec_dir}
```

I would prefer using the standard macros here:
```suggestion
%{_cross_libexecdir}/amazon-ecs-agent/managed-agents
```

I'd expect rpm to emit a "file listed twice" warning here because it seems like these macros would expand to:
```
%{_cross_libexecdir}/amazon-ecs-agent/managed-agents
%{_cross_libexecdir}/amazon-ecs-agent/managed-agents/execute-command
```